pydrumscore/export_files/META-INF/container.xml

```diff
@@ -0,0 +1,9 @@
+<container>
+  <rootfiles>
+    <rootfile full-path="King Nothing.mscx" />
```

Comment:
See my other comment, this line would be removed and instead be created on the spot.